vocabularies/report-detail-type.ttl

```diff
@@ -35,14 +35,22 @@ rptdt:address a skos:Concept ;
     skos:prefLabel "Address"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/report-detail-type> .
 
-rptdt:comments a skos:Concept ;
-    skos:altLabel "Description"@en,
-        "Summary"@en,
+rptdt:activities a skos:Concept ;
+    skos:altLabel "Summary"@en,

```

Comment:
I like this concept now, but I would remove Summary - that belongs as an alt label in Comments

Reply:
Moved